src/v2/core.ts

```diff
@@ -49,10 +49,10 @@ export interface TriggerAnnotation {
 }
 
 /**
- * A CloudEvent is a cross-platform format for encoding a serverless event.
+ * A CloudEventBase is the base of a cross-platform format for encoding a serverless event.
  * More information can be found in https://github.com/cloudevents/spec
  */
-export interface CloudEvent<T> {
+export interface CloudEventBase<T> {
```

Comment:
hm maye this no longer needs to be exported?

Reply:
removed the export